client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -975,7 +975,7 @@ Undocumented.prototype.readTags = function( fn ) {
 Undocumented.prototype.readTagPosts = function( query, fn ) {
 	var params = omit( query, 'tag' );
 	debug( '/read/tags/' + query.tag + '/posts' );
-	params.apiVersion = '1.2';
+	params.apiVersion = '1.3';
 	this.wpcom.req.get( '/read/tags/' + encodeURIComponent( query.tag ) + '/posts', params, fn );
```

Comment:
i think what's being shown here is fine... do you think this isn't what i'm asking for?